scheduling-manager-workflows/src/main/java/com/amazonaws/blox/schedulingmanager/deployment/handler/types/DeploymentInput.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.schedulingmanager.deployment.handler.types;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonDeserialize(builder = DeploymentInput.DeploymentInputBuilder.class)
+@Builder
+@Value
+public class DeploymentInput {
+  private final String name;
+  private final String account;
+
+  @JsonPOJOBuilder(withPrefix = "")
```

Comment:
Does this generate an additional builder class, if it's used alongside the lombok `@Builder` annotation? Or does it reuse that one?

Reply:
Not sure actually, but it's the same name the lombok builder uses so I'm assuming there's only one builder. I'll check.

Reply:
Yeah, that sounds right. Unfortunate that the boilerplate for that is a bit opaque :(